doc/source/user_guide/io.rst

```diff
@@ -2477,6 +2490,10 @@ A few notes on the generated table schema:
     * For ``MultiIndex``, ``mi.names`` is used. If any level has no name,
       then ``level_<i>`` is used.
 
+* When using a ``DataFrame`` containing a ``Series`` backed by a used-defined
+  ``ExtensionArray``, the generated JSON will contain an extra ``extDtype``
+  key under the respective ``fields`` array element. While this key is not standard
+  it enables roundtripping for custom types (e.g. ``read_json(df.to_json(orient="table"), orient="table")``).
```

Comment:
The above 4 lines should appear after
```
.. _Table Schema: https://specs.frictionlessdata.io/table-schema/
```

or possibly deleted, because if you move the other text there, it becomes somewhat duplicated.